### ISSUE TITLE
how to tune the RPN  anchor and ROI paprameters for training?

### ISSUE THREAD

**Issue by lunasdejavu** [matterport/Mask_RCNN#1733]
I tried to cut the license plate from images by this project.
the image size is about 1920x1080  around 8000 for training 1000 for validation 1000 for test
the 
my training configuration is like:
```
class LPConfig(Config):
    NAME = "Licenseplatedetection"
    GPU_COUNT = 1
    IMAGES_PER_GPU = 2
    BACKBONE = "resnet50"
    # Number of classes (including background)
    NUM_CLASSES = 1 + 1  # background + 1 shapes
    IMAGE_MIN_DIM = 512
    IMAGE_MAX_DIM =  1024
    qq = 4
    RPN_ANCHOR_SCALES = (8 * qq, 16 * qq, 32 * qq, 64 * qq, 128 * qq) #(32, 64, 128, 256, 512) 
    STEPS_PER_EPOCH = 1000  
    VALIDATION_STEPS = 100
    LEARNING_RATE= 0.0005
    MAX_GT_INSTANCES = 6
    DA_NUMBER_PER_IMAGE=20
```
the result is like this https://imgur.com/a/KynaHDJ
the loss function by tensorboard https://imgur.com/a/UgU2dLi
but the result is not very well since the license plates are not segmented rightly and too many weird false positive situation.
since the dataset can't be wrong and  so does this project 
can anyone tell me how to tune the parameters?

**Comment by Hunain-Gillani** [comment]
How did you changed the learning rate? I tried editing in config.py file and it didn't work.

**Comment by lunasdejavu** [comment]
> How did you changed the learning rate? I tried editing in config.py file and it didn't work.
no one can answer your question if you don't post your code here

**Comment by tobenan** [comment]
hi,why do you set RPN_ANCHOR_SCALES = (8 * qq, 16 * qq, 32 * qq, 64 * qq, 128 * qq) like this?why not  (32, 64, 128, 256, 512)? is this because you object is square?
